change/@ni-nimble-components-d849f476-f620-4021-bf50-d5f1258be733.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
The tooltip has some aggressive padding in Firefox on macOS (I didn't test other browsers). This looks vaguely familiar...we may have solved this in a previous tooltip PR but I can't recall how.

For reference, I used [the fractional width column story](https://60e89457a987cf003efc0a5b-hjirkftres.chromatic.com/?path=/story/incubating-table-column-configuration--fractional-width-column) to test it (struggled to find a case where the columns would ellipsize before the table hit its min width).

<img width="397" alt="image" src="https://github.com/ni/nimble/assets/10500124/e93f5747-93c0-4798-b67a-534e64dfab1c">